compose.yml

```
@@ -440,6 +440,25 @@ services:
       - ./environment/splatoon.env
       - ./environment/splatoon.local.env
 
+  minecraft-wiiu:
+    build: ./repos/minecraft-wiiu
+    depends_on:
+      - account
+    restart: unless-stopped
+    ports:
+      # Go delve debugger
+      - 127.0.0.1:2352:2350
```

Looks like the ports are swapped.
```suggestion
      - 127.0.0.1:2350:2352
```

---

the port 2350 is already binded by the splatoon delve debugger.

---

Oh, whoops. Your original order was right, my bad!